README.md

```diff
@@ -52,7 +58,12 @@ author.save(function(err){
 ```
 
 ```rb
-
+@author = Author.create(author_params)
+respond_to do |format|
+  if @author.save
+    format.html { redirect_to: authors_path }
+  end
+end
```

Comment:
`create` includes saving the author to the database. The Ruby equivalent would entail calling `.new` and then `.save`